src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.BoolValueSet.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    using static BinaryOperatorKind;
```

Comment:
>using static BinaryOperatorKind; [](start = 4, length = 32)

nit: why nest this using?

Reply:
Because doing it outside requires full qualification, which is more distracting than informative.

---
In reply to: [381016472](https://github.com/dotnet/roslyn/pull/41503#discussion_r381016472) [](ancestors = 381016472)